.travis.yml

```diff
@@ -1,39 +0,0 @@
----
-language: python
-python: "2.7"
-
-# Use the new container infrastructure
-sudo: false
-
-# Install ansible
-addons:
-  apt:
-    packages:
-    - python-pip
-
-install:
-  # Install ansible
-  - pip install ansible ansible-lint yamllint os-cloud-config
-
-  # Check ansible version
-  - ansible --version
-
-  # Create ansible.cfg with correct roles_path
-  - printf '[defaults]\nroles_path=../' >ansible.cfg
-
-  # Compensate for repo name being different to the role
-  - ln -s ansible-role-os-manila-mount/ ../stackhpc.os-manila-mount
-
-  # Obtain custom yamllint rules for Ansible
-  - curl -O https://raw.githubusercontent.com/ansible/galaxy/devel/galaxy/importer/linters/yamllint.yaml
-
-script:
-  # Basic role syntax check
-  - ansible-playbook tests/test.yml -i tests/inventory --syntax-check
-  # Ansible lint
-  - ansible-lint tests/test.yml
-  # YAML lint
-  - yamllint -c yamllint.yaml .
```

Comment:
I would like to keep the CI though, is it easy to add the github actions in here... I guess we have a standard way to do this now?

Reply:
To be clear, I really mean the linting, I think that would be enough without that syntax check thingy.

Reply:
Oh I missed this. Can I add it back in later!